components/raftstore-v2/src/operation/command/admin/raft_log_gc.rs

```diff
@@ -0,0 +1,271 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! This module contains processing logic of the following:
+//!
+//! # `RaftLogGc` and `EntryCacheEvict` ticks
+//!
+//! On region leader, periodically compacts useless Raft logs from the
+//! underlying log engine, and evicts logs from entry cache if it reaches memory
+//! limit.
+//!
+//! # `CompactLog` command
```

Comment:
Maybe we should rename the command to `GcLog`. I think keeping two names are confusing. Better always use gc log.

Reply:
But isn't that a name from kvproto? Might as well change log_gc to log_compact..

Reply:
Yes. To minimize the impact for now, we can just change everything within TiKV.

Reply:
Then I think we should unify to compact instead, it involves fewer changes and is consistent with raft paper.

Reply:
That works too.